engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/document/DocumentPredicates.kt

```diff
@@ -0,0 +1,24 @@
+package graphql.nadel.enginekt.document
+
+import graphql.normalized.VariablePredicate
+
+class DocumentPredicates {
```

Comment:
No need to fix, but just FYI you could've just made this an object i.e.

```
object DocumentPredicates {
  val jsonPredicate
  val allVariablesPredicate
}
```

And the behavior would be the same